build_tools/github/test_source.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+set -x
+
+python -m venv test_env
+source test_env/bin/activate
+
+python -m pip install dist/*.tar.gz
+python -m pip install pytest pandas
+
+# Run the tests on the installed source distribution
+mkdir tmp_for_test
+cd tmp_for_test
+
+pytest --pyargs sklearn
```

Comment:
Actually I have better hypothesis: pytest might be looking up the `setup.cfg` file in the parent folder. If we want to ignore it:

```suggestion
# Run the tests on the installed source distribution
mkdir ../tmp_for_test
cd ../tmp_for_test

pytest --pyargs sklearn
```

Reply:
The original problem seems to be solved, but a new error is raised:

```
ValueError: option names {'--skip-network'} already added
```

Are the network tests skipped by default in source distributions?

Reply:
Copying the `conftest.py` file seems to be the problem: https://stackoverflow.com/questions/55117947/valueerror-option-names-already-added-pytest. I will delete such statement and check if the tests run.

Reply:
Applying the previous changes raises the original problem. This is quite strange 😕.

Reply:
The issue have been solved. Moving up two levels is the right way to create a sibling folder because actually the path the `scikit-learn` source code is `$HOME/scikit-learn/scikit-learn` due to the way that `actions/checkout` works.